src/main/groovy/com/ullink/gradle/opencover/OpenCover.groovy

```diff
@@ -49,6 +55,14 @@ class OpenCover extends ConventionTask {
         openCoverExec
     }
 
+    def getReportGeneratorConsole() {
+        def reportGenerator = 'reportgenerator-'+ reportGeneratorVersion
+        Path path = Paths.get(project.gradle.gradleUserHomeDir.toString(), 'caches', 'reportgenerator', reportGenerator)
+        File reportGeneratorExec = new File(path.toString(), "ReportGenerator.exe")
+
```

Comment:
should add a validation that the file exists, similar as in getOpenCoverConsole

Reply:
Done